Makefile

```diff
@@ -74,10 +74,10 @@ test_e2e:
 	go test ./e2e -count 1 -test.parallel=8 -v $(args)
 
 test_e2e_ci:
-	go test ./e2e -count 1 -v -test.parallel=6 -json 2>&1 | tee /tmp/gotest.log | gotestfmt
+	go test ./e2e -v -test.parallel=7 -json 2>&1 | tee /tmp/gotest.log | gotestfmt
```

Comment:
Why remove the `-count` flags? Were they interfering with something? We usually use this to disable the go test cache, which isn't helpful for our E2E tests.